### DISCUSSION TITLE
Regarding using IPFS

### DISCUSSION THREAD

**Discussion by mktetts** [aptos-labs/aptos-developer-discussions#434] [General]
Hello team, for the hackathon, should we use Irys storage or any other decentralized storage like ipfs, pinata ?

**Comment by gregnazario** [comment]
I think that's totally up to you and the tradeoffs associated.

IPFS you have to make sure you don't unpin it, and arweave you need the token associated to store it.  Both should be perfectly fine.

**Reply by mktetts** [reply]
Thanks for reply, and another question is should our finished app is deployed is deployed in cloud like aws, vercel etc, 
or it is fine with finished code with github repo, video and instructions to run the project?
